notebooks/amazon_sagemaker_and_chroma_for_qa.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p>We're. not using <a href="https://docs.haystack.deepset.ai/v2.0/docs/pypdftodocument" target="_blank">PyPDFToDocument</a> here so let's update the description like this:</p><p>"Our indexing pipeline will preprocess the provided Wikipedia pages by cleaning and splitting it into chunks before storing them in <a href="https://docs.haystack.deepset.ai/v2.0/docs/chroma-document-store" target="_blank">ChromaDocumentStore</a>. "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>